terra2/assetlist.json

```diff
@@ -510,4 +510,52 @@
         }
       }
     ]
+  },
+{
+  "denom_units": [
+    {
+      "denom": "ibc/A356EC90DC3AE43D485514DA7260EDC7ABB5CFAA0654CE2524C739392975AD3C",
+      "exponent": 0,
+      "aliases": [
+        "wstETH"
+      ]
+    },
+    {
+      "denom": "factory/neutron1ug740qrkquxzrk2hh29qrlx3sktkfml3je7juusc2te7xmvsscns0n2wry/wstETH",
+      "exponent": 6
+    }
+  ],
+  "type_asset": "ics20",
+  "base": "ibc/A356EC90DC3AE43D485514DA7260EDC7ABB5CFAA0654CE2524C739392975AD3C",
+  "name": "Canonical Wrapped stETH",
+  "display": "wstETH",
+  "symbol": "wstETH",
+  "traces": [
+    {
+      "type": "ibc",
+      "counterparty": {
+        "chain_name": "neutron",
+        "base_denom": "factory/neutron1ug740qrkquxzrk2hh29qrlx3sktkfml3je7juusc2te7xmvsscns0n2wry/wstETH",
+        "channel_id": "channel-25"
```

Comment:
Can we get this ibc connection registered first, please?

Reply:
I don't know what it is or how to register it... I just IBC bridge wstETH from Terra to Neutron and looked at the TX, and saw that the dst_channel was 25.

Reply:
in the _IBC directory there are ibc connections registered. We need one for neutron<>terra2 before we can add assets that originate over these ibc connections

Reply:
Added the connection, see my most recent commit.